connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroLogicalTypeSuite.scala

```diff
@@ -436,7 +436,7 @@ abstract class AvroLogicalTypeSuite extends QueryTest with SharedSparkSession {
       val ex = intercept[SparkException] {
         spark.read.format("avro").load(s"$dir.avro").collect()
       }
-      assert(ex.getErrorClass == "FAILED_READ_FILE")
+      assert(ex.getErrorClass.startsWith("FAILED_READ_FILE"))
```

Comment:
Why don't compare to `FAILED_READ_FILE.NO_HINT`?

Reply:
We may keep adding more sub error classes to `FAILED_READ_FILE` and I want the tests to be stable.